### ISSUE TITLE
load_model uses compat.v1 BatchNorm

### ISSUE THREAD

**Issue by mhutti1** [tensorflow/tensorflow#39474]
**System information**
- Have I written custom code : Yes
- OS Platform and Distribution: Ubuntu
- TensorFlow installed from (source or binary): Source
- TensorFlow version (use command below): 2.2.0.dev20200304
- Python version: 3.6.9

**Describe the current behavior**
``` tf.keras.models.load_model``` loads models in with ```tf.compat.v1.keras.layers.BatchNormalization```

**Describe the expected behavior**
It should load models with tf.keras.layers.BatchNormalization

**Standalone code to reproduce the issue**
See https://github.com/tensorflow/model-optimization/issues/386

**Comment by alanchiao** [comment]
As I mentioned in the other issue, this may have been fixed as of 2.2.0. Isaac will double check and if it's not resolved, this issue should be escalated.

**Comment by mhutti1** [comment]
Yep, upgrading to 2.2.0 fixed it. Thanks for all the work :)
